tests/UnsafeInterfacesTest.php

```diff
@@ -48,4 +63,10 @@ public function testAlwaysValidChild() {
     $xhp = <html>{$x}<body /></html>;
     $this->assertEquals('<html>foo<body></body></html>', $xhp->toString());
   }
+
+  public function testUnsafeAttribute(): void {
+    $escaped = new ExampleUnsafeAttribute('foo && bar');
+    $xhp = <script onload={$escaped} />;
+    $this->assertEquals('<script onload="foo && bar"></script>', $xhp->toString());
```

Comment:
can you add a counterexample of the expected result when *not* using XHPUnsafeRenderable?